pons/routing/router.py

```diff
@@ -98,10 +103,28 @@ def on_scan_received(self, msg: pons.Message, remote_id: int):
             # self.log("DUP PEER: %d" % remote_id)
 
     def on_peer_discovered(self, peer_id):
-        self.log("peer discovered: %d" % peer_id)
+        #self.log("peer discovered: %d" % peer_id)
+        self.netsim.event_manager.on_peer_discovery(self.my_id, peer_id)
 
     def on_msg_received(self, msg: pons.Message, remote_id: int):
-        self.log("msg received: %s from %d" % (msg, remote_id))
+        self.netsim.routing_stats['relayed'] += 1
+        if not self.is_msg_known(msg):
+            self.remember(remote_id, msg)
+            msg.hops += 1
+            self.store_add(msg)
```

Comment:
what if i want to model a router without storage, only directly forwarding and then forgetting the bundle?
here, i always have a persistent store. 
yes, i could ignore it but then the question is does it have performance impact for evaluations?